storage/container_storage_interface/persistent-storage-csi-cinder.adoc

```diff
@@ -17,6 +17,9 @@ To create CSI-provisioned PVs that mount to OpenStack Cinder storage assets, {pr
 
 * The _OpenStack Cinder CSI driver_ enables you to create and mount OpenStack Cinder PVs.
 
+For {product-title}, automatic migration from OpenStack Cinder in-tree to the CSI driver is available as a Technology Preview (TP) feature.
+With migration enabled, volumes provisioned using the existing in-tree storage class are automatically migrated to use the OpenStack Cinder CSI storage class. For more information, see xref:persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration feature].
```

Comment:
**OpenStack Cinder CSI storage class -> OpenStack Cinder CSI driver** will be better.